README.md

```diff
@@ -27,7 +27,7 @@ NGINX Gateway Fabric uses [NGINX Agent](https://github.com/nginx/agent) to confi
 1. [Get started using a kind cluster](https://docs.nginx.com/nginx-gateway-fabric/get-started/).
 2. [Install](https://docs.nginx.com/nginx-gateway-fabric/install/) NGINX Gateway Fabric.
 3. Deploy various [examples](examples).
-4. Read our [How-to guides](https://docs.nginx.com/nginx-gateway-fabric/how-to/).
+4. Follow instructions for common use cases such as [routing](https://docs.nginx.com/nginx-gateway-fabric/traffic-management/basic-routing) and [securing](https://docs.nginx.com/nginx-gateway-fabric/traffic-security/securing-backend/) traffic, or [monitoring](https://docs.nginx.com/nginx-gateway-fabric//monitoring/) your cluster.
```

Comment:
For security, can we just link to the top level folder? Both docs are relevant, arguably the other one is more important.